addons/spreadsheet/static/src/list/list_actions.js

```diff
@@ -14,19 +15,19 @@ export const SEE_RECORD_LIST = async (position, env) => {
         .map(astToFormula)
         .map((arg) => env.model.getters.evaluateFormula(sheetId, arg));
     const listId = env.model.getters.getListIdFromPosition(position);
-    const { model } = env.model.getters.getListDefinition(listId);
+    const { model, actionXmlId } = env.model.getters.getListDefinition(listId);
     const dataSource = await env.model.getters.getAsyncListDataSource(listId);
     const recordId = dataSource.getIdFromPosition(evaluatedArgs[1] - 1);
     if (!recordId) {
         return;
     }
-    await env.services.action.doAction({
+    await navigateTo(env, actionXmlId, {
         type: "ir.actions.act_window",
         res_model: model,
         res_id: recordId,
         views: [[false, "form"]],
         view_mode: "form",
-    });
+    }, { viewType: "form" });
```

Comment:
What's the difference between `view_mode` and `viewType` again ?

Reply:
view mode is deprecated